src/main/java/com/laserfiche/repository/api/clients/impl/ApiClient.java

```diff
@@ -27,15 +31,16 @@ public ApiClient(String baseUrl, UnirestInstance httpClient) {
                 .getObjectMapper();
     }
 
-    protected String mergeMaxSizeIntoPrefer(int maxSize, String prefer) {
+    protected static String mergeMaxSizeIntoPrefer(int maxSize, String prefer) {
```

Comment:
moved all static functions to the `ApiUtilClient` class except for `sendRequestWithRetry` and the new created `parseDictionaryResponse()` function since they are more closely related to the ApiClient and could be frequently changed in the future